Tools/AssetLogAnalyzer/AssetLogAnalyzer.py

```diff
@@ -0,0 +1,464 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+

```

Comment:
Might be worth adding a top-of-file comment on what this prints out, and how to use it.